mlflow/pipelines/steps/predict.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import os
+import time
+from typing import Dict, Any
+
+from mlflow.pipelines.cards import BaseCard
+from mlflow.pipelines.step import BaseStep
+from mlflow.pipelines.utils.execution import get_step_output_path
+from mlflow.pipelines.utils.step import get_pandas_data_profile
+
+
+_logger = logging.getLogger(__name__)
+
+
+_INPUT_FILE_NAME = "dataset.parquet"
```

Comment:
This should be the output file name of the `preprocess` step?

Reply:
I added a comment but I think you will eventually want to import this from the preprocessing step.  I'll leave it for now tho.